Flutter/calendar/drag-drop.md

```diff
@@ -86,7 +86,7 @@ void dragStart(AppointmentDragStartDetails appointmentDragStartDetails) {
 [targetResource](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/AppointmentDragUpdateDetails/targetResource.html) - Get the resource details.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="9 10" %}
```

Comment:
Check 9 is needed or not and highlight method also

Reply:
Method highlighted.